lib/i18n_fixes.rb

```diff
@@ -67,28 +67,3 @@ def locale
     end
   end
 end
-
```

Comment:
I don't think it's a globalize change, I think it's that Rails 4.2 doesn't accidentally hold on to the cached value the way that 4.1 did. I suspect we ended up patching Globalize to work around a Rails 4.1 bug.

(Also as `@attributes_cache` has been removed in 4.2, the patch crashes the app post upgrade)

The Globalize code looks pretty similar between the versions:
- [4.0.3](https://github.com/globalize/globalize/blob/v4.0.3/lib/globalize/active_record/instance_methods.rb#L22-L59)
- [5.0.1](https://github.com/globalize/globalize/blob/v5.0.1/lib/globalize/active_record/instance_methods.rb#L22-L57)